go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -31,6 +31,10 @@ const (
 	egressFirewallAppliedCorrectly = "EgressFirewall Rules applied"
 	egressFirewallAddError         = "EgressFirewall Rules not correctly added"
 	aclDeleteBatchSize             = 1000
+	// create time (which is the upper bound of how much time an update can take) for 20K ACLs
+	// (gress ACL were used for testing as the ones that have the biggest number of ExternalIDs)
+	// is ~4 sec, which is safe enough to not exceed 10 sec transaction timeout.
+	aclUpdateBatchSize = 20000
```

Comment:
I am confused on how you are referring to create time of ACLs when we are doing here is updating PGs.